### ISSUE TITLE
Unclear Docs - mocker.patch - no attribute assert_called_once_with

### ISSUE THREAD

**Issue by ahurlburt** [pytest-dev/pytest-mock#282]
I can't seem to find in the documentation anywhere how to assert mock function created with mocker.patch is called

Example:

```
def test(self, x):
   return True

mock_a = mocker.patch('SomeClass.test', test)

SomeClass.some_test_method_that_calls_mocked_method()

mock_a.assert_called_once_with("a")
```

I get an error stating function object has no attribute 'assert_called_once_with'

The only example in the docs I see use this method but there is no example setting up the mock like I have. Am I missing something? Am I correct that https://docs.python.org/3/library/unittest.mock.html these functions should be supported? Honestly I find the docs pretty hard to follow as to what types of assertions are actually supported...

**Comment by nicoddemus** [comment]
Hi @ahurlburt,

Can you provide a full reproducible case? I can't from the example tell what might be wrong.

> Honestly I find the docs pretty hard to follow as to what types of assertions are actually supported...

`pytest-mock` is mostly a thin wrapper around `unittest.mock`, so we support the same methods.

**Comment by danielpassy** [comment]
An example: the mock clearly registered a function called, however assert_called_with_once returns an error.
![image](https://user-images.githubusercontent.com/20033745/172169560-2d006cb2-f74e-4789-8643-e47ef20f23ab.png)

**Comment by nicoddemus** [comment]
Closing due to inactivity
